async/models.py

```diff
@@ -243,6 +244,17 @@ def execute():
                     "Exception is %s."
                     "Trace is %s",
                 self.id, self.scheduled, errors, self.priority, repr(exception), format_exc())
+            try:
+                info = {
+                    'job' : self.id,
+                    'num_errors' : errors,
+                    'new_priority': self.priority
```

Comment:
I guess we should also add repr(exception) in the info?

Reply:
The captureException() API from the sentry sdk already does the job of collecting the exception info and send it accross. So, I don't think there is any need to do that. We should just manually add any contextual information that is not available in the exception message and traceback.